pyproject.toml

```diff
@@ -12,10 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-[metadata]
+[project]
 name = "lightning"
-author = "Lightning-AI et al."
-url = "https://github.com/Lightning-AI/lightning"
+dynamic = ["version"]
+authors = [{name = "Lightning-AI et al."}]
+readme = "README.md"
```

Comment:
This is depending on the created package

Reply:
Sorry, are you referring to the readme specifically? I can remove that field if that's what you mean.